crates/astria-sequencer/src/service/info.rs

```diff
@@ -36,28 +32,119 @@ use tendermint::{
 use tower::Service;
 use tower_abci::BoxError;
 use tracing::{
-    debug,
     instrument,
-    warn,
     Instrument,
 };
 
-use crate::{
-    accounts::query::QueryHandler,
-    state_ext::StateReadExt,
-};
+#[derive(Clone, Copy, Debug)]
+pub(crate) struct AbciCode(u32);
+impl AbciCode {
+    pub(crate) const INTERNAL_ERROR: Self = Self(3);
+    pub(crate) const INVALID_PARAMETER: Self = Self(2);
+    pub(crate) const OK: Self = Self(0);
+    pub(crate) const UNKNOWN_PATH: Self = Self(1);
```

Comment:
nit but can you order these :p also are these from somewhere? if so can you add a comment about that

Reply:
I did order them, but rustfmt decided that it preferred them in alphabetic order. 😭

Reply:
omg :| that's annoying lol

Reply:
I forgot to add: no, these are application defined, right? So we could do some bikeshedding on the naming haha

Reply:
true, we could prefix them with something, up to you lol